Migrate legacy Mark licence for supp. billing test

Cruikshanks opened the pull request:
We're onto the last folder of legacy tests; `internal/billing`. No surprise there are tests that have little to do with billing and would be better located elsewhere. The first of these is `mark-licence-for-supplementary-billing.spec.js`.

It is in the billing folder and is named as a test to mark a licence for supplementary billing. But having reviewed the test its primary concern is to submit a return against a licence. Marking it for supplementary billing is a small step it tags on the end!

So, we bring this into our `internal/returns` folder. What we also realised was our existing `submit.cy.js` test was actually using the record receipt option in returns. So, a rename frees up 'submit' for the legacy test we're migrating.

Finally, whilst we were doing this we knew that `internal/charge-information/sroc/journey.cy.js` also results in a licence being marked for supplementary billing. So, we tweaked the end of that test to include confirmation this happens.

fayaz0211 commented:
> Looks good to me, merge it if the tests pass :)

The test passes.merging the branch now.

> Looks good to me, merge it if the tests pass :)

Cruikshanks committed:
Migrate legacy Mark licence for supp. billing test

We're onto the last folder of legacy tests; `internal/billing`. No surprises there are tests that have little to do with billing and would be better located elsewhere. The first of these is `mark-licence-for-supplementary-billing.spec.js`.

It is in the billing folder and is named as a test to mark a licence for supplementary billing. But having review the test it primary concern is to submit a return against a licence. Marking the it for supplementary billing is a small step it tags on the end!

So, we bring this in to our `internal/returns` folder. What we also realised was our existing `submit.cy.js` test was actually using the record receipt option in returns. So, a rename frees up 'submit' for the legacy test we're migrating.

Finally, whilst we were doing this we knew that `internal/charge-information/sroc/journey.cy.js` also results in a licence being marked for supplementary billing. So, we tweaked the end of that test to include confirmation this happens.
